### PR TITLE
fixed sample exchange bug

### PR DESCRIPTION
**Pull request by EDsCODE**
https://trello.com/c/0hK3Eoe9/255-exchange-to-agent-sms-none

-exchange example was incorrectly rounding

### CODE REVIEW

**Comment by EDsCODE** [comment]
[exchange to agent sms (None)](https://trello.com/c/0hK3Eoe9/255-exchange-to-agent-sms-none)

File: app/server/utils/ussd/token_processor.py [hunk 1]
```diff
@@ -198,7 +198,7 @@ def fetch_exchange_rate(user: User):
 
         exchange_limit = rounded_dollars(TokenProcessor.get_default_exchange_limit(default_limit, user))
         exchange_rate = round_to_sig_figs(exchange_rate_full_precision, 3)
-        exchange_sample_value = rounded_dollars(exchange_rate_full_precision * float(1000))
```
Comment (EDsCODE) [inline app/server/utils/ussd/token_processor.py]: for documentation purposes: the rounding function that rounds money to dollars is meant to take an input thats in cents (for example: rounding a balance of 5000 cents into 50 dollars)

File: app/server/utils/misc.py [hunk 2]
```diff
@@ -38,6 +38,10 @@ def round_to_decimals(amount, decimals=2):
 
 
 def rounded_dollars(amount):
+    """
+    :param amount: money amount in cents (e.g. 2500)
+    :return: rounded dollars as 25.00 (if <1000)
+    """
```
Comment (tristanhcole) [inline app/server/utils/misc.py]: @EDsCODE :)

File: app/server/utils/ussd/token_processor.py (hunk 1 above)
Comment (tristanhcole) [reply app/server/utils/ussd/token_processor.py]: done

**Pull request merged**